src/main/groovy/com/vherasymenko/avro/encoder/core/CourseInstallEncoderService.groovy

```diff
@@ -41,7 +42,7 @@ class CourseInstallEncoderService implements CourseInstallEncoderPort {
 
     @Override
     void handleEvent( String event, String processChannelName ) {
-        def schema = CourseInstall.classSchema
+        def schema = new Schema.Parser().parse( new File( 'src/main/avro/course_install.avsc' ) )
```

Comment:
So I was a little confused by what you meant in the overview about parsing from a file, and I see it here, so I guess I'm still a little puzzled? Maybe I am missing something obvious but I still could use a little more explanation why we aren't retrieving the data from the schema.

I'm guessing it's due to the mangling of the types?